site/content/includes/installation/install-gateway-api-resources.md

```diff
@@ -7,12 +7,14 @@ docs: "DOCS-1438"
 To install the Gateway API resources, run the following:
 
 ```shell
-kubectl apply -f https://github.com/kubernetes-sigs/gateway-api/releases/download/v1.1.0/standard-install.yaml
+kubectl kustomize "https://github.com/nginxinc/nginx-gateway-fabric/config/crd/gateway-api/standard?ref=v1.2.0" | kubectl apply -f -
```

Comment:
will it work on Windows? piping?

Reply:
I just tried it with https://github.com/PowerShell/PowerShell and it works